dts/msm8916-samsung.dts

```diff
@@ -34,4 +34,15 @@
 			i2c-address = <0x14>;
 		};
 	};
+
+	j5nlte {
+	       model = "Samsung J5N LTE (EUR)";
```

Comment:
Please make sure your editor is using tabs.

Also, I know I did it this way for a5ulte for some reason, but I think a proper model string is better here. Something like `Samsung Galaxy J5 (SM-J500FN)`, as long as it fits on the line in the lk2nd fastboot screen.